### PR TITLE
The new way

### PR DESCRIPTION
**Pull request by stephenplusplus**

### CODE REVIEW

**Comment by jmdobry** [comment]
What's the blocker here?

**Comment by stephenplusplus** [comment]
I'm not all the way through the repo migration doc yet, so... I don't know :) But I haven't merged those other PRs yet, which feed into this repo-migration branch.

**Comment by stephenplusplus** [comment]
@jmdobry I'm getting an error while running the samples `test`:

```
  × logs » should write a log entry
  √ sinks » should create a sink (2.8s)
  √ sinks » should get a sink (1.4s)
  √ sinks » should list sinks (1.5s)
  √ sinks » should update a sink (1.7s)
  √ quickstart » should log an entry (8.4s)
  √ logs » should write a simple log entry (8.2s)
  √ sinks » should delete a sink (1.6s)
  √ logs » should write using winston (8.7s)
  √ logs » should write using bunyan (8.7s)

  1 test failed [12:08:32]

  logs » should write a log entry
  C:\Users\sawch\dev\nodejs-logging\samples\system-test\logs.test.js:1

   1: 'use strict';
   2:

  Difference:

  - ''
  + 'Wrote to nodejs-docs-samples-test-c18b359e-7d40-4944-83bc-3d50f4eda3d9'
```

Any ideas?

File: samples/fluent.js [hunk 1]
```diff
@@ -42,15 +42,15 @@ var report = function (err, req) {
...
+        responseStatusCode: 500,
+      },
+    },
   };
   structuredLogger.emit('errors', payload);
 };
 
 // Handle errors (the following uses the Express framework)
-app.use(function (err, req, res, next) {
+app.use(function(err, req, res) {
```
Comment (jmdobry) [inline samples/fluent.js]: Turns out the `next` argument is required, it's how Express knows this middleware function is an error-handler middleware. Need to add `next` back and disable the linter.

**Comment by jmdobry** [comment]
Hmmm, the logs.test.js system test passes for me locally, so I'm not sure what's going on for you. You're on windows?

File: samples/fluent.js (hunk 1 above)
Comment (stephenplusplus) [reply samples/fluent.js]: Done.

**Comment by stephenplusplus** [comment]
Yep, I'm on Windows. The cause of all problems with Node 😢

**Comment by stephenplusplus** [comment]
Another Windows/AppVeyor/gRPC problem:

> Error: Cannot find module 'C:\projects\nodejs-logging\node_modules\grpc\src\node\extension_binary\node-v46-win32-ia32\grpc_node.node'

**Comment by stephenplusplus** [comment]
Node v4 errors with gRPC on Circle CI (https://circleci.com/gh/googleapis/nodejs-logging/3?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link):

#### npm install Log Output
```
> grpc@1.6.0 install /root/project/node_modules/google-gax/node_modules/grpc
> node-pre-gyp install --fallback-to-build --library=static_library

node-pre-gyp info it worked if it ends with ok
node-pre-gyp info using node-pre-gyp@0.6.36
node-pre-gyp info using node@4.8.4 | linux | x64
node-pre-gyp info check checked for "/root/project/node_modules/google-gax/node_modules/grpc/src/node/extension_binary/node-v46-linux-x64/grpc_node.node" (not found)
node-pre-gyp http GET https://storage.googleapis.com/grpc-precompiled-binaries/node/grpc/v1.6.0/node-v46-linux-x64.tar.gz
node-pre-gyp http 200 https://storage.googleapis.com/grpc-precompiled-binaries/node/grpc/v1.6.0/node-v46-linux-x64.tar.gz
node-pre-gyp info install unpacking grpc_node.node
node-pre-gyp info tarball done parsing tarball
[grpc] Success: "/root/project/node_modules/google-gax/node_modules/grpc/src/node/extension_binary/node-v46-linux-x64/grpc_node.node" is installed via remote
node-pre-gyp info ok 
```

#### Error
```
Error: Cannot find module '/root/project/node_modules/@google-cloud/common-grpc/node_modules/grpc/src/node/extension_binary/node-v46-linux-x64/grpc_node.node'
```

**Comment by stephenplusplus** [comment]
Fixed the issue above by forcing an `npm rebuild` in the Circle CI configuration.

**Pull request merged**